src/libsyntax/feature_gate.rs

```diff
@@ -1136,6 +1136,15 @@ impl<'a> Visitor<'a> for PostExpansionVisitor<'a> {
                     },
                     _ => {}
                 }
+
+                match defaultness {
```

Comment:
What I meant here is that this `match` should be replaced with a `if let`. `hir` here was my mistake, sorry.